.github/blunderbuss.yml

```diff
@@ -0,0 +1,7 @@
+assign_issues:
+  - sofisl
+  - bcoe 
+assign_prs:
+  - sofisl
+  - bcoe  
+  
```

Comment:
Not to be too nit picky, but it looks like there are a few extra lines at the end of this file (🤔 out loud, I wonder if we could make our linter catch this).